lib/compiler-finder.js

```diff
@@ -225,6 +225,7 @@ export class CompilerFinder {
             objdumper: props('objdumper', ''),
             objdumperType: props('objdumperType', ''),
             intelAsm: props('intelAsm', ''),
+            hasAsmDocs: props('hasAsmDocs', true),
```

Comment:
Can we name this after the other "features", so "supportsAsmDocs"?

Reply:
you're right, thanks